app/res/xml/provider_paths.xml

```diff
@@ -2,4 +2,5 @@
 <paths>
     <external-files-path name="all_external_files" path="."/>
     <external-files-path name="external_temp" path="temp/external/"/>
+    <external-path name="external_files" path="." />
```

Comment:
what does this do ?

Reply:
`external-files-path` gives content uri for `storage/emulated/0/data/org.commcare.dalvik/` directory and `external-path` gives content uri for `storage/emulated/0/` directory. So I added this to get share contentUris of external files like the ones in downloads or dcim folder.